packages/client/auth-core/src/contexts/authlogin.tsx

```diff
@@ -4,46 +4,44 @@ import AuthModal from "@/components/AuthModal";
 import { CrossmintServiceFactory } from "@/services/CrossmintService";
 import { type ReactNode, createContext, useContext, useEffect, useMemo, useState } from "react";
 
+import { Crossmint } from "@crossmint/common-sdk-base";
+
+export function getCachedJwt(): string | undefined {
+    if (typeof document === "undefined") {
+        return undefined; // Check if we're on the client-side
+    }
+    const crossmintSession = document.cookie.split("; ").find((row) => row.startsWith("crossmint-session"));
```

Comment:
is `crossmint-session` a const we should extract out so it's easier to refactor later?

Reply:
fixed